provider/sidecar.go

```diff
@@ -10,21 +10,34 @@ import (
 	"strings"
 	"time"
 
-	fsnotify "gopkg.in/fsnotify.v1"
-
 	"github.com/BurntSushi/toml"
 	"github.com/Nitro/sidecar/catalog"
 	"github.com/Nitro/sidecar/service"
 	"github.com/containous/flaeg"
 	"github.com/containous/traefik/log"
 	"github.com/containous/traefik/safe"
 	"github.com/containous/traefik/types"
+	fsnotify "gopkg.in/fsnotify.v1"
 )
 
 const (
-	method = "wrr"
-	weight = 0
-	sticky = false
+	method               = "wrr"
+	weight               = 0
+	sticky               = false
+	maxConnExtractorFunc = "request.host"
```

Comment:
That's just there until we finally agree to have this setting configurable via TOML along with the actual limit. In our case "request.host" makes sense, but other people might want to change it. It's used [here](https://github.com/Nitro/traefik/blob/master/vendor/github.com/vulcand/oxy/utils/source.go#L24) and it can be set to "client.ip" or "request.host" or "request.header.<header_name>". I think it's best if we allow users to change it via the config file.